src/sprites/rendered-target.js

```diff
@@ -215,9 +215,9 @@ class RenderedTarget extends Target {
      */
     static get VIDEO_STATE () {
         return {
-            'OFF': 'off',
-            'ON': 'on',
-            'ON-FLIPPED': 'on-flipped'
+            OFF: 'off',
+            ON: 'on',
+            ON_FLIPPED: 'on_flipped'
```

Comment:
I don't think we want this change, since we display `on-flipped` in the dropdown and don't want it to read `on_flipped`.